velox/core/SimpleFunctionMetadata.h

```diff
@@ -536,6 +537,44 @@ class SimpleFunctionMetadata : public ISimpleFunctionMetadata {
     return s;
   }
 
+  std::string toString(const std::string& name) const final {
+    std::stringstream sig;
```

Comment:
do not abbreviate: sig -> signature

It is strange that we do not include the result type here. Let's add it.

You seem to be building signatures twice: once using logical types and once again using physical types. If that's the case, then let's name variables to match: logicalSignature and physicalSignature. Let's make sure both signatures include argument types and return type.